deepreg/dataset/load.py

```diff
@@ -33,6 +33,7 @@ def get_data_loader(data_config: dict, mode: str) -> (DataLoader, None):
         return None
     if isinstance(data_dir_paths, str):
         data_dir_paths = [data_dir_paths]
+    data_dir_paths = list(map(os.path.expanduser, data_dir_paths))
```

Comment:
maybe add a comment saying

"replace ~ in the paths by user's home directory"

otherwise I wonder is there any other paths in the config

Reply:
> maybe add a comment saying
> 
> "replace ~ in the paths by user's home directory"
> 
> otherwise I wonder is there any other paths in the config

not i can find in the config file, only other places would be the CLI... let me check